lib/CleantalkSP/Common/Helpers/HelperIP.php

```diff
@@ -264,12 +264,13 @@ public static function get( $ip_type_to_get = 'real', $headers = array() )
      * @param mixed $ip
      *
      * @return string|bool
+     * 
      */
     public static function validate($ip)
     {
         if(!$ip) return false; // NULL || FALSE || '' || so on...
         if(filter_var($ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV4) && $ip !== '0.0.0.0') return 'v4';  // IPv4
-        if(filter_var($ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV6) && self::v6_reduce($ip) !== '0::0') return 'v6';  // IPv6
+        if(filter_var($ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV6) && self::v6_reduce($ip) !== '::') return 'v6';  // IPv6
```

Comment:
Why do we need this?
Is there some problem with 0::0?

Reply:
method self::v6_reduce returns :: when ip == 0:0:0:0:0:0:0:0

Reply:
ok